### ISSUE TITLE
tdrop with alacritty not working

### ISSUE THREAD

**Issue by EdgardoArriagada** [noctuid/tdrop#129]
when I do execute 
```shell
$ tdrop alacritty
```

it returns:
```shell
error: Found argument '-d' which wasn't expected, or isn't valid in this context

USAGE:
    alacritty [FLAGS] [OPTIONS]

For more information try --help
Error: Exceeded timeout of 10 seconds waiting for program.
```
which is a similar output of
```shell
$ alacritty -d
```
_or any other unlisted flag for `alacritty`_

I'm on Ubuntu 20.04

**Comment by EdgardoArriagada** [comment]
I just commented the `if` statement at line 665

```zsh
  # if [[ $program == alacritty ]]; then
    # prevent alacritty from resizing the terminal to 80x24
    # program_command+=(-d 0 0)
  # fi
```
then, I recompiled again

```shell
$  (cd ~/tdrop && sudo make install)
```

and it seems to work fine now, I excecute alacritty like this:

```shell
$  tdrop -h 100% alacritty
```

**Comment by maybeetree** [comment]
A workaround that doesn't require recompiling would be to symlink `/bin/alacritty` to a file with a different name and call that new name from tdrop

**Comment by CodePurble** [comment]
The symlink hack does work. Don't know why though.

**Comment by c02y** [comment]
I'm using Manjaro, alacritty in Manjaro official repo just gets upgraded directly from `v0.6.0` to `v.0.7.1`, and I got the exact same problem.

tdrop script has
```
 if [[ $program == alacritty ]]; then
    # prevent alacritty from resizing the terminal to 80x24
    program_command+=(-d 0 0)
 fi
```
which uses the `-d` option from alacritty, the `-d` option is supported in `v0.6.0` but it is gone in `v0.7.1`. This is the problem.

If using other strings other than exact "alacritty" such as "/bin/alacritty" or rename the binary in PATH and use the renamed binary, the above code inside if-loop wouldn't be executed, then it is OK.

tdrop just needs an update.

**Comment by noctuid** [comment]
Is the resize issue back though, or does alacritty maintain the specified size? Did another flag replace -d?

**Comment by CodePurble** [comment]
Alacritty does maintain the specified size, no issues there after the workaround is applied. The `-d` option has a replacement, I think it has been dropped.

**Comment by c02y** [comment]
I believed this issue can be closed since the latest commit of tdrop fixed it or you can fix it by yourself accord to previous replies.

**Comment by EdgardoArriagada** [comment]
Closed since it is solved in 89938c1483821c3924ca4e298d38f85f63961781
